docs/reference/builder.md

```diff
@@ -1291,6 +1291,68 @@ The `STOPSIGNAL` instruction sets the system call signal that will be sent to th
 This signal can be a valid unsigned number that matches a position in the kernel's syscall table, for instance 9,
 or a signal name in the format SIGNAME, for instance SIGKILL.
 
+## HEALTHCHECK
+
+The `HEALTHCHECK` instruction has two forms:
+
+* `HEALTHCHECK [OPTIONS] CMD command` (check container health by running a command inside the container)
```

Comment:
Having `CMD` here is really confusing.

I think it would be better have simply `HEALTHCHECK <cmd>` that would be the default health checking command, kind of like `ENTRYPOINT` but for healthchecks.